dash-pipeline/dockerfiles/dev-notes/design-slimmer-bmv2.md

```diff
@@ -35,10 +35,12 @@ run-switch:
 chris@chris-z4:~/chris-DASH/DASH/dash-pipeline$ make run-all-tests
```

Comment:
Do me a favor and delete this dev-notes directory, it's obsolete.

Reply:
done